other-components/html/index.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * LaravelCollective/html
+ *
+ * @source https://github.com/LaravelCollective/html
+ * @contributor https://github.com/dead23angel
+ */
+
+require_once 'vendor/autoload.php';
+
+use Collective\Html\HtmlBuilder;
+use Collective\Html\FormBuilder;
+use Illuminate\Config\Repository as Config;
+use Illuminate\Container\Container;
+use Illuminate\Events\Dispatcher;
+use Illuminate\Http\Request;
+use Illuminate\Filesystem\Filesystem;
+use Illuminate\Routing\Router;
+use Illuminate\Routing\UrlGenerator;
+use Illuminate\Session\SessionManager;
+
+$app           = new Container();
```

Comment:
This is a lot of work just to use a package that helps you echo forms. Is there any reason you would want to use LaravelCollective/HTML instead of something like [Bootform](https://github.com/adamwathan/bootforms)? Maybe there is, just curious.